<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingFilterHorizontalLinear</class>
 <widget class="QDialog" name="SettingFilterHorizontalLinear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxStrength">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>117</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>147</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain / (Distance from Center)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLeftSide">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Left side</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_swf_64.PNG</normaloff>J:/検査画面icon/flash_swf_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRightSide">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Right side</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Strength</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxWholeImage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Adapt in whole image</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbLayer1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer 1</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxGamma">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>187</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixel Gamma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
